web/twa-vis-platform/code/src/ui/interaction/tree/layer/layer-tree.tsx

```diff
@@ -145,6 +139,11 @@ function collectIDs(layers: DataLayer[]): string {
  * @returns {string} the icon name
  */
 function getIcon(layers: DataLayer[], icons: IconSettings): string {
+  const overridelayer: DataLayer = layers.find(layer => layer.definition?.layerTreeIconOverride);
```

Comment:
Checking if this is a new property? If so, please document this in the README so that we can keep track